app/deliver_grant_funding/templates/deliver_grant_funding/grant_setup/ggis_required_info.html

```diff
@@ -0,0 +1,38 @@
+{% if grant %}
+  {% extends "deliver_grant_funding/grant_base.html" %}
+  {% set active_item_identifier = "details" %}
+  {% set back_href = back_link_href or url_for("deliver_grant_funding.grant_change_ggis", grant_id=grant.id) %}
+{% else %}
+  {% extends "deliver_grant_funding/base.html" %}
+  {% set active_item_identifier = "grants" %}
+  {% set back_href = url_for("deliver_grant_funding.grant_setup_ggis") %}
+{% endif %}
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+
+{% set page_title = "You need to have a GGIS reference number before you can add this grant" %}
+
+{% block beforeContent %}
+  {{
+    govukBackLink({
+      "text": "Back",
+      "href": back_href
+    })
+  }}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <span class="govuk-caption-l">Set up the grant</span>
+      <h1 class="govuk-heading-l">You need to have a GGIS reference number before you can add this grant</h1>
+      <p class="govuk-body">
+        You'll need to complete the
+        <a href="https://forms.office.com.mcas.ms/pages/responsepage.aspx?id=EGg0v32c3kOociSi7zmVqBUKhC0CqZtGmIj1YcYa53xUNTFRWkRXQ1ZJUEJMOTg1UllGWEpCNDQ4NSQlQCN0PWcu&route=shorturl" target="_blank" class="govuk-link">
+          Pipeline Grants Scheme Form (opens in new tab)
+        </a>
+        and the Finance Process and Data team will issue your grant with a reference number.
+      </p>
+      <p class="govuk-body">If you've lost your GGIS reference number, contact the team at ggis@communities.gov.uk.</p>
```

Comment:
nit: Can we move this email address into config please? I think just nice to centralise things like that there 👀 

Another nit: can we make this a `mailto:` link? Unlinked does match the prototype but I think linking it is reasonable (Design System team do it for example: https://design-system.service.gov.uk/get-in-touch/)